specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1048 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra cluster."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The managed Cassandra cluster is being updated. Poll the provided operation for completion."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/repair": {
+      "post": {
+        "operationId": "CassandraCluster_RequestRepair",
+        "description": "Request that repair begin on this cluster as soon as possible.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraRepair": {
+            "$ref": "./examples/CosmosDBManagedCassandraRepair.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The repair operation will begin as soon as possible."
+          },
+          "409": {
+            "description": "Repair is disabled for this cluster, so no repair request can be accepted. To enable it, set the cluster's repairEnabled property to true."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/fetchNodeStatus": {
+      "post": {
```

Comment:
>   "post": { [](start = 3, length = 12)

Is this change made in our Service ? Ideally we should not merge swagger until the change at our Service side is deployed in Prod.

Reply:
None of these exist in prod yet anyway. They're going out in hotfixes at the moment, and we're trying to coordinate all this stuff for our public preview date.